great_expectations/compatibility/docstring_parser.py

```diff
@@ -1,17 +1,20 @@
 from __future__ import annotations
 
+import warnings
+
 from great_expectations.compatibility.not_imported import NotImported
 
 DOCSTRING_PARSER_NOT_IMPORTED = NotImported(
     "docstring_parser is not installed, please 'pip install docstring-parser'"
 )
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", category=DeprecationWarning)
```

Comment:
Maybe add a comment explaining this filter.

Reply:
Done